tests/client_test.go

```diff
@@ -14,17 +14,56 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package trello
+package tests
 
 import (
+	"os"
 	"testing"
 
 	"github.com/VojtechVitek/go-trello"
+
+	. "github.com/franela/goblin"
```

Comment:
I'm not too familiar with any, but for [another project](https://github.com/fiatjaf/summadb) I have here, I have tried reading about **testify** a little, but don't know if I like it or not. About **ginkgo**, I've written a lot of tests with it, but then I discovered it is fundamentally against running sequentials tests, among other quibbles I have with it, so I switched to **goblin**.

If you prefer testify and if you think it can run tests sequentially, please setup some tests (or rewrite these tests I have put up) so I can see how it is done and add others. Does that make sense?